torch_np/_ndarray.py

```diff
@@ -504,6 +504,13 @@ def asarray(a, dtype=None, order="K", *, like=None):
     return array(a, dtype=dtype, order=order, like=like, copy=False, ndmin=0)
 
 
+def ascontiguousarray(a, dtype=None, order="K", *, like=None):
+    arr = array(a, dtype=dtype, order=order, like=like, copy=False, ndmin=0)
```

Comment:
Probably marginally better to call `asarray` here forwarding the call to `asarray` unless we plan to implement `order="C"` in terms of this function.

Reply:
...and avoid an extra copy because `copy=True`. Done.